SU-HSDO/suhumsci
# STN-192 : root font size

# [STN-192](https://sparkbox.atlassian.net/secure/RapidBoard.jspa?rapidView=80&modal=detail&selectedIssue=STN-192) READY FOR REVIEW

## Summary
Update root font size to 10px and adjust styles where we currently use rems.

## Need Review By (Date)
asap

## Urgency
high 🔥

## Steps to Test
1. In the CLI, run `npm run test` and verify all of our tests pass.
2. Run `npm start` to compile the CSS files.
3. Go to http://swshumsci.suhumsci.loc
4. Review several different pages to confirm font-sizes, spacing, heights, widths, etc display as expected. For reference: [Figma](https://www.figma.com/file/toNZnFLadtpNpG5dPzNGWz/⚙%EF%B8%8F-Colorful-Theme---Working?node-id=1408%3A1313)
- [ ] Horizontal list
- [ ] Vertical list
- [ ] Exposed filters
    * Buttons
    * Inputs
- [ ] Events list
- [ ] People
- [ ] Primary nav
- [ ] Secondary nav
- [ ] Logo lock-up
- [ ] Global footer

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
